code/modules/organs/subtypes/unathi.dm

```diff
@@ -0,0 +1,46 @@
+#define BLOOD_REGEN_PER_SEC   0.07
+#define STAMINA_REGEN_PER_SEC 5
+
+// Unathi Blood Regeneration organ
+/obj/item/organ/internal/suphreans
+	name = "suphreans"
+	icon_state = "generic"
+	desc = "An organ that exists within Sinta'Unathi that grants them enhanced blood regeneration. It is found near the liver."
+	gender = PLURAL
+	organ_tag = BP_SUPHREANS
+	parent_organ = BP_GROIN
+	min_bruised_damage = 25
+	min_broken_damage = 45
+	max_damage = 70
+	relative_size = 6
+	toxin_type = CE_HEPATOTOXIC
+	var/last_process
+
+/obj/item/organ/internal/suphreans/process()
+	..()
+	if(!owner)
+		return
+	owner.add_blood_simple(BLOOD_REGEN_PER_SEC * ((world.time - last_process) / 10))
```

Comment:
Is this intended to be 0.007?

You're adding [0.07 * (1 / 10)] blood, which is [0.07 * 0,1] -> 0,007

Reply:
nope, math is hard